srcpkgs/grub-btrfs/files/grub-btrfs/run

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+exec 2>&1
+[ -r conf ] && . ./conf
+
+if [ -d "${SNAPSHOTS_PATH}" ]
+then
+	exec wendy ${OPTS} -m 960 -w "${SNAPSHOTS_PATH}" bash -c \
```

Comment:
I don't think `bash` should be hardcoded here, but it's guaranteed to be pulled in by the package, so leaving as is for now.

Reply:
@ericonr that command is taken from upstream's systemd service. I don't think they will switch to POSIX sh, so it would be fine

Reply:
Upstream might use bash in a systemd service, but they aren't providing a runit service, so we should adopt Void idioms when adapting our own. There is nothing specific to bash, so /bin/sh is the way to go here.

Reply:
That sounds reasonable. I will make the change